packages/ods/src/components/checkbox/src/components/ods-checkbox/ods-checkbox.tsx

```diff
@@ -80,22 +107,36 @@ export class OdsCheckbox {
     });
   }
 
+  private onInvalidEvent(event: Event): void {
+    // Remove the native validation message popup
+    event.preventDefault();
+    event.stopPropagation();
+
+    // Enforce the state here as we may still be in pristine state (if the form is submitted before any changes occurs)
+    this.isInvalid = true;
+  }
+
   render(): FunctionalComponent {
     return (
-      <Host class="ods-checkbox">
+      <Host class="ods-checkbox"
+        disabled={ this.isDisabled }>
```

Comment:
isn't the required also necessary?

Reply:
It's the readonly but the checkbox hasn't readonly